src/client/src/serviceWorker.js

```diff
@@ -30,8 +30,7 @@ export function register(config) {
     }
```

Comment:
Is there a reason this file is JavaScript and not TypeScript?

Reply:
Also, should this be in a PWA-specific "platform", as opposed to the main codebase? Not sure tbh. It's probably here for a reason.

Reply:
JS because it's not compiled as part of the our source. It's in the public directory (same as the create-react-app default one).